<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroEditorDialog</class>
 <widget class="QDialog" name="HeroEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Hero</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <layout class="QGridLayout" name="gridLayout_form">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="placeholderText">
        <string>Hero name...</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_res_name">
       <property name="text">
        <string>Resource Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_player">
       <property name="placeholderText">
        <string>Player name...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_temp_hp">
       <property name="text">
        <string>Temp HP</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_resource_name">
       <property name="placeholderText">
        <string>Spell points, stamina, luck, etc...</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_res_values">
       <property name="text">
        <string>Resource (current / max)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_hp_current">
       <property name="text">
        <string>Current HP</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_hp_max">
       <property name="text">
        <string>Max HP</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_resource">
       <item>
        <widget class="QSpinBox" name="spinBox_resource_current">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_res_slash">
         <property name="text">
          <string>/</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_resource_max">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_class">
       <property name="placeholderText">
        <string>e.g. Fighter, Mage, Healer...</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="lineEdit_conditions">
       <property name="placeholderText">
        <string>Comma-separated conditions (e.g. Poisoned, Stunned)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinBox_hp_max">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_class">
       <property name="text">
        <string>Class / Role</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBox_hp_current">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_level">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_level">
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBox_temp_hp">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_conditions">
       <property name="text">
        <string>Conditions</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_player">
       <property name="text">
        <string>Player</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_faction">
       <property name="text">
        <string>Faction / Tag</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_faction">
       <property name="placeholderText">
        <string>Heroes, Allies, NPC, etc...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_notes_public">
     <property name="text">
      <string>Notes (public)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_notes_public">
     <property name="placeholderText">
      <string>Visible to players...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_notes_gm">
     <property name="text">
      <string>Notes (GM)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_notes_gm">
     <property name="placeholderText">
      <string>Private GM notes...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HeroEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HeroEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
